### DISCUSSION TITLE
How to enable "Enable SQL Server Failover support" feature in TE3

### DISCUSSION THREAD

**Discussion by carloscantu22** [TabularEditor/TabularEditor3#821] [🙋♀️TE3 Q&A] [answered]
Hello,

I'm trying to find a way to enable an equivalent option to the "Enable SQL Server Failover support" connections advanced option available in Power BI desktop while opening a model from database using TE3. Can you please assist?

This option is available under Get Data/SQL Server as shown  in the attached file.

Thank you!
![Enable SQL Server Failover Support](https://user-images.githubusercontent.com/67908362/218748034-6d161324-4a1c-45fa-825b-d54ca8e695fb.PNG)

**Comment by mlonsk** [comment, marked as answer]
You need to set the property Multi Subnet Failover to True when you create your data source.
It can be found under advanced -> Source has shown bellow. 

![image](https://user-images.githubusercontent.com/11939114/220152616-fcbcc53b-6fcf-495c-8c94-3775ebc25053.png)

**Reply by carloscantu22** [reply]
Thank you, I wasn't able to find this property for some of our datasets when connecting to the database, so as a workaround we added the folllowing parameters to the connection string, ApplicationIntent=ReadOnly;MultiSubnetFailover=True
